test-loop-tests/src/tests/spice.rs

```diff
@@ -158,8 +158,159 @@ fn test_spice_chain() {
         .shutdown_and_drain_remaining_events(Duration::seconds(20));
 }
 
+#[cfg(feature = "test_features")]
+#[test]
+#[cfg_attr(not(feature = "protocol_feature_spice"), ignore)]
+fn test_spice_chain_with_missing_chunks() {
+    use crate::utils::account::{
+        create_validators_spec, rpc_account_id, validators_spec_clients_with_rpc,
+    };
+    use crate::utils::node::TestLoopNode;
+
+    init_test_logger();
+    let accounts: Vec<AccountId> =
+        (0..100).map(|i| format!("account{}", i).parse().unwrap()).collect_vec();
```

Comment:
nit: `create_account_ids<100>("account").to_vec()`

Reply:
create_account_ids doesn't seem to work that way:
https://github.com/near/nearcore/blob/7583d6aa5071d9d43f789a4e1cc24a658e068438/test-loop-tests/src/utils/account.rs#L8-L10

In any case, I changed the code to use create_account_id instead in 9c0ab7dcd